docs/agent/version-support-policy.mdx

```diff
@@ -44,7 +44,8 @@ Extended support beyond LTS is available to customers on a commercial basis. It
 
 | Support Status | Release | Release Date | Latest | Active Support | End of Life |
 | -------------- | ------- | ------------ | ------ | -------------- | ----------- |
-| Active         | 3.12x   | 2024-06-27   | 3.12.0 | 2025-06-27     | 2025-08-27  |
+| Active         | 3.13x   | 2024-07-18   | 3.13.0 | 2025-07-18     | 2025-09-18  |
```

Comment:
Note: we need to update the version support policy doc as well as part of the agent release process.